manifests/base/kustomization.yaml

```diff
@@ -15,355 +15,647 @@ resources:
   - jupyter-rocm-pytorch-notebook-imagestream.yaml
   - jupyter-rocm-tensorflow-notebook-imagestream.yaml
 
-commonLabels:
-  opendatahub.io/component: "true"
-  component.opendatahub.io/name: notebooks
 
 configMapGenerator:
-  - name: notebooks-parameters
-    env: params.env
-  - name: notebook
-    env: commit.env
+  - envs:
+      - params.env
+    name: notebooks-parameters
+  - envs:
+      - commit.env
+    name: notebook
 generatorOptions:
   disableNameSuffixHash: true
-vars:
-  - name: odh-minimal-notebook-image-n
-    objref:
+
+configurations:
+  - params.yaml
+  - commit.yaml
+labels:
+  - includeSelectors: true
+    pairs:
+      component.opendatahub.io/name: notebooks
+      opendatahub.io/component: "true"
+replacements:
+  - source:
+      fieldPath: data.odh-minimal-notebook-image-n
       kind: ConfigMap
       name: notebooks-parameters
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.odh-minimal-notebook-image-n
-  - name: odh-minimal-notebook-image-n-1
-    objref:
+      version: v1
+    targets:
+      - fieldPaths:
+          - spec.tags.0.from.name
+        select:
+          group: image.openshift.io
+          kind: ImageStream
+          name: jupyter-minimal-notebook
+          version: v1
+  - source:
+      fieldPath: data.odh-minimal-notebook-image-n-1
       kind: ConfigMap
       name: notebooks-parameters
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.odh-minimal-notebook-image-n-1
-  - name: odh-generic-data-science-notebook-image-n
-    objref:
+      version: v1
+    targets:
+      - fieldPaths:
+          - spec.tags.1.from.name
```

Comment:
Chatbot suggests this can be written

```
        fieldPaths:
          - spec.template.spec.containers.[name=myapp].image
```

So instead of the weird `.1.`, there may be other possibilities. Imo not very useful here, but may be useful for notebook-controller, maybe.

Reply:
Hm, so you mean we would go with something like:
```
      - fieldPaths:
          - spec.tags.[name=2025.1].from.name
```

This could be a bit more clearer when reading and would be more resistant to errors in case somebody will be changing the order of the tags in the ImageStreams, true. I shall consider your proposal.

I'm just wondering whether we won't introduce a bit more necessary changes once we will be introducing a new image versions (e.g. 2025.2), which will lead to update also this part :thinking:

Reply:
I updated this PR with the changes that allows this to be build by the latest kustomize version out there as of today (5.6.0). And just realized about this your proposal. I'm not against it, but let's check the others opinion. We can also do it as a separate change if needed. WDYT?

Reply:
> I'm not against it, but let's check the others opinion. We can also do it as a separate change if needed. WDYT?

certainly a separate change!